dnf-behave-tests/common/output.py

```diff
@@ -54,6 +54,17 @@ def handle_reposync(expected, found, dnf5_mode):
     return expected, found
 
 
+@behave.then("{dnf_version:dnf_version} exit code is")
```

Comment:
It seems the `{dnf_version:dnf_version}` pattern is being used before it is registered below (`behave.register_type(dnf_version=parse_dnf_version)`) so its not working.